### ISSUE TITLE
AuthStateMissing at /complete/google-oauth2/

### ISSUE THREAD

**Issue by PatrickJazzMea** [gravitylow/OpenCourse#5]
Went to login and this happened after entering valid credentials.

**Comment by gravitylow** [comment]
This is probably not the technical answer you were expecting but this randomly started happening the other day and I have legitimately no idea why. So I've been fiddling with it - I think it's a bug with cookies not being set. We'll see what happens. It's a good thing I don't have a deadline on this haha.

**Comment by gravitylow** [comment]
In hindsight, that probably should have been more obvious. In a recent change I upgraded the site to use SSL/TLS and made a settings change (to SESSION_COOKIE_SECURE, which hasn't been committed due to testing) which prevents cookies from being sent over non-HTTPS sessions. I've removed that so it's fixed now and will either be allowing insecure cookies or forcing all connections to be over HTTPS.

Thanks for the report!
